src/browser-frontend/model/domain-page-meta-model.js

```diff
@@ -19,6 +19,7 @@ const LOAD_STATES = {
   INITIAL: 'initial',
   CONNECTING: 'connecting',
   LOADING: 'loading',
+  ANIMATED: 'animated',
```

Comment:
This definitely isn't some kind of "load state" related to the page domain model.